disnake/permissions.py

```diff
@@ -198,9 +213,24 @@ def all_channel(cls: Type[P]) -> P:
         .. versionchanged:: 2.3
             Added :attr:`start_embedded_activities` permission.
         """
-        return cls(0b1111110110110011111101111111111101010001)
+        guild_specific_perms = {
+            "administrator",
+            "ban_members",
+            "change_nickname",
+            "kick_members",
+            "manage_emojis",
+            "manage_guild",
+            "manage_nicknames",
+            "moderate_members",
+            "view_audit_log",
+            "view_guild_insights",
+        }
+        instance = cls.all()
+        instance.update(**dict.fromkeys(guild_specific_perms, False))  # type: ignore
```

Comment:
```suggestion
        instance.update(**dict.fromkeys(guild_specific_perms, False))
```
doesn't seem to be required

Reply:
https://github.com/DisnakeDev/disnake/pull/286/commits/6cc1ab5c497b37df099f6f3dc209764798e55935